ArduPlane/geofence.cpp

```diff
@@ -91,24 +91,31 @@ void Plane::set_fence_point_with_index(Vector2l &point, unsigned i)
 void Plane::geofence_load(void)
 {
     uint8_t i;
+    const char* geofence_error_str = "Geofence error,";
 
     if (geofence_state == nullptr) {
         uint16_t boundary_size = sizeof(Vector2l) * max_fencepoints();
         if (hal.util->available_memory() < 100 + boundary_size + sizeof(struct GeofenceState)) {
             // too risky to enable as we could run out of stack
-            goto failed;
+            g.fence_action.set(FENCE_ACTION_NONE);
+            gcs().send_text(MAV_SEVERITY_WARNING,"%s low on memory", geofence_error_str);
```

Comment:
miss a very important space after warning comma ! (missing on all warning)

Reply:
Would be better to put the space in the `geofence_error_str` as it's then common :)